BindFamilyLockBonusActivationActivity/index.ts

```diff
@@ -0,0 +1,53 @@
+import { AzureFunction, Context } from "@azure/functions";
+import { readableReport } from "italia-ts-commons/lib/reporters";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
+import { BonusActivation } from "../generated/models/BonusActivation";
+import { CosmosDbDocumentCollection, toBaseDoc } from "../services/cosmosdb";
+import { generateFamilyUID } from "../utils/hash";
+
+const index: AzureFunction = async (_: Context, input: unknown) => {
+  const decoded = CosmosDbDocumentCollection.decode(input);
+  if (decoded.isLeft()) {
+    throw Error(
+      `BindFamilyLockBonusActivationActivity: cannot decode input [${readableReport(
+        decoded.value
+      )}]`
+    );
+  }
+  const documents = decoded.value.reduce(
```

Comment:
can we use `t.readonlyArray(BonusActivation).decode` instead ?

Reply:
With `t.readonlyArray(BonusActivation).decode` we skip all the records even if only one doesn't match the format. Is it right?